packages/compass-connect/src/stores/index.js

```diff
@@ -307,6 +315,15 @@ const Store = Reflux.createStore({
    * validate instead the existing connection object.
    */
   async onConnectClicked() {
+    const { connectionModel } = this.state;
+    const [major, minor] = process.env.HADRON_APP_VERSION.split('.');
+    const trackEvent = {
+      compass_version: `${major}.${minor}`,
```

Comment:
`compass_version` is automatically added to all events, no need to worry about it here. You are making the very good point that the tracking doc only says to use major.minor, not the full version, so I’ve added a commit to #2560 to fix that there, thank you :slightly_smiling_face: